site/content/docs/manifest/worker-service.en.md

```diff
@@ -97,4 +100,104 @@ Required. The service this SNS topic is exposed by. Together with the topic name
+
+If you want to use Fargate Spot capacity to run your services, you can specify a number under the `spot` subfield:
+```yaml
+count:
+  spot: 5
+```
+
+<div class="separator"></div>
+
+Alternatively, you can specify a map for setting up autoscaling:
+```yaml
+count:
+  range: 1-10
+  cpu_percentage: 70
+  memory_percentage: 80
+  queue_delay:
+    acceptable_latency: 10m
+    msg_processing_time: 250ms
+```
+
+<span class="parent-field">count.</span><a id="count-range" href="#count-range" class="field">`range`</a> <span class="type">String or Map</span>  
+You can specify a minimum and maximum bound for the number of tasks your service should maintain, based on the values you specify for the metrics.
+```yaml
+count:
+  range: n-m
+```
+This will set up an Application Autoscaling Target with the `MinCapacity` of `n` and `MaxCapacity` of `m`.
+
+Alternatively, if you wish to scale your service onto Fargate Spot instances, specify `min` and `max` under `range` and then specify `spot_from` with the desired count you wish to start placing your services onto Spot capacity. For example:
+
+```yaml
+count:
+  range:
+    min: 1
+    max: 10
+    spot_from: 3
+```
+
+This will set your range as 1-10 as above, but will place the first two copies of your service on dedicated Fargate capacity. If your service scales to 3 or higher, the third and any additional copies will be placed on Spot until the maximum is reached.
+
+<span class="parent-field">range.</span><a id="count-range-min" href="#count-range-min" class="field">`min`</a> <span class="type">Integer</span>  
+The minimum desired count for your service using autoscaling.
+
+<span class="parent-field">range.</span><a id="count-range-max" href="#count-range-max" class="field">`max`</a> <span class="type">Integer</span>  
+The maximum desired count for your service using autoscaling.
+
+<span class="parent-field">range.</span><a id="count-range-spot-from" href="#count-range-spot-from" class="field">`spot_from`</a> <span class="type">Integer</span>  
+The desired count at which you wish to start placing your service using Fargate Spot capacity providers.
+
+<span class="parent-field">count.</span><a id="count-cpu-percentage" href="#count-cpu-percentage" class="field">`cpu_percentage`</a> <span class="type">Integer</span>  
+Scale up or down based on the average CPU your service should maintain.
+
+<span class="parent-field">count.</span><a id="count-memory-percentage" href="#count-memory-percentage" class="field">`memory_percentage`</a> <span class="type">Integer</span>  
+Scale up or down based on the average memory your service should maintain.
+
+<span class="parent-field">count.</span><a id="count-queue-delay" href="#count-queue-delay" class="field">`queue_delay`</a> <span class="type">Integer</span>   
+Scale up or down based to maintain an acceptable backlog per task.  
```

Comment:
Hmmm. "based"? 🤔

Reply:
Would it be more relatable to say "scale up or down to maintain an acceptable latency"? I feel like scaling base on "backlog per task" is a mean to the end of achieving an acceptable latency.

Reply:
Ty both! I updated it like this:
> Scale up or down to maintain an acceptable queue latency by tracking against the acceptable backlog per task.

I wanted to mention the "acceptable backlog per task" here as it will help them understand why we are explaining it below.